src/components/app-header/app-header.css

```diff
@@ -0,0 +1,29 @@
+.app-header {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 0 0 20px;
+}
+
+.search-form {
+  margin-bottom: 20px;
+  padding: 10px;
+  text-decoration: none;
+  font-size: 16px;
+}
+
+.type-box {
+  height: 5rem;
```

Comment:
https://www.w3schools.com/cssref/css_units.asp

rem is relative to the font size of root element. Use px unless you are sure what you are doing